honeybadgermpc/betterpairing.py

```diff
@@ -144,22 +165,25 @@ def duplicate(self):
     def projective(self):
         return self.pyg1.projective()
 
+    def pair_with(self, other):
+        fq12 = PyFq12()
+        self.pyg1.py_pairing_with(other.pyg2, fq12)
+        return GT(fq12)
+
     @staticmethod
     def one():
-        out = PyG1()
-        out.zero()
-        return G1(out)
+        return G1(PyG1().zero())
```

Comment:
This is the problem - `PyG1.zero()` zeroes out the element in place and returns `None`, rather than returning the `PyG1` interest.